cpp/oneapi/dal/test/engine/dataframe_common.cpp

```diff
@@ -219,6 +219,36 @@ class dataframe_builder_action_allocate : public dataframe_builder_action {
     std::int64_t column_count_;
 };
 
+class dataframe_builder_action_read_external_dataset : public dataframe_builder_action {
+public:
+    explicit dataframe_builder_action_read_external_dataset(std::string dataset)
+            : dataset_(dataset) {}
+
+    std::string get_opcode() const override {
+        return fmt::format("read_external_dataset({})", dataset_);
+    }
+
+    dataframe_impl* execute(dataframe_impl* df) const override {
+        delete df;
+
+        const char* dataset_root_ptr = std::getenv("DATASETSROOT");
```

Comment:
Not sure, but we have `DATASETS_ROOT`, but not `DATASETSROOT`, please double check.

Reply:
'DAAL_DATASETS'